proto/spire/api/registration/registration.proto

```diff
@@ -32,6 +32,24 @@ message UpdateEntryRequest {
     spire.common.RegistrationEntry entry = 1;
 }
 
+// A type that represents pagination for list responses
+message Pagination {
+    string token = 1;
+    int32 page_size = 2;
+}
+
+//
+message ListAllEntriesRequest {
+    string token = 1;
```

Comment:
Should this instead have a `Pagination` field?

Reply:
Added `Pagination` in `ListAllEntriesRequest`.